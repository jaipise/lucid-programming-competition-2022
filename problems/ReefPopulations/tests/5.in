10 5
16 16 20 16 16 15 5 5 16 10
2 8
5 10
10 10
6 8
3 7