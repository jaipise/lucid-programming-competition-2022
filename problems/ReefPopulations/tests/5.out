11
5
10
15
30
